ci/e2e-tests/suite-cleanup.sh

```diff
@@ -39,7 +39,21 @@ while :; do
         break
     fi
 
-    <<< "$ids" timeout 30s xargs az resource delete --ids >/dev/null
+    # A bug in the latest az CLI causes az resource delete to use an API
+    # version not supported by the cloud for DPS instances. For now, we
+    # will manually specify the API version when deleting DPS instances.
+    #
+    # Ref: https://github.com/Azure/azure-cli/issues/20263
+    readarray -t ids <<< "$ids"
+
+    for resource in "${ids[@]}"
```

Comment:
This will be many times slower because it'll delete resources one at a time.

What I'd suggested was to filter out the DPS ID in one place, run `az resource delete` on all the other resources first, and then on the DPS ID with the API version parameter.

Reply:
Fixed

Reply:
Thanks. Is the timeout for the suite-cleanup stages sufficient? Or do you need to double it like I'd said on Teams?

Reply:
Yes, it's sufficient. I haven't noticed it timing out.